packages/angular_devkit/build_angular/src/dev-server/ssl_spec.ts

```diff
@@ -102,8 +102,8 @@ describe('Dev Server Builder ssl', () => {
 
     const overrides = {
       ssl: true,
-      sslKey: '../ssl/server.key',
-      sslCert: '../ssl/server.crt',
+      sslKey: 'ssl/server.key',
+      sslCert: 'ssl/server.crt',
```

Comment:
Did this test not work before?

Reply:
Yeah, wrong paths

Reply:
Perviously, wrong paths errors where swallowed.